src/libraries/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -272,6 +272,13 @@ public static IEnumerable<object[]> Parse_Valid_TestData()
             yield return new object[] { (567.89).ToString(), defaultStyle, null, 567.89 };
             yield return new object[] { (-567.89).ToString(), defaultStyle, null, -567.89 };
             yield return new object[] { "1E23", defaultStyle, null, 1E23 };
```

Comment:
I do not see tests here (nor in SingleTests.cs or DecimalTests.cs) for values ending a in decimal separator, at least not ending in period. eg `yield return new object[] { "123.", defaultStyle, null, 123 };`

Reply:
Maybe I'm looking in the wrong place as not only do I not see anything testing parsing ending in `.` but also not starting with `.` or even with `0.`

Reply:
That's actually odd, all of the parsing tests I ported back in https://github.com/dotnet/corefx/pull/33355/files are missing

Reply:
Found them. I had forgotten we moved them when we brought the UTF8 formatter/parser online.

The tests live here:
* https://github.com/dotnet/runtime/blob/master/src/libraries/Common/tests/System/RealParserTestsBase.cs
* https://github.com/dotnet/runtime/blob/master/src/libraries/Common/tests/System/RealFormatterTestsBase.cs

Reply:
@pgovind, it might be good to add some new tests here as well

Reply:
did you end up concluding we have the tests I mentioned above?

Reply:
Ah you commented before I had a chance. Nope. We actually don't have those tests. I opened another issue for it: https://github.com/dotnet/runtime/issues/48119.